test/e2e/tests/test_helper.py

```
@@ -167,6 +171,50 @@ def _gateway_url():
+    the ephemeral curl pod before running curl.
+
+    Otherwise falls back to the service-account CA already mounted in the pod.
+
+    Returns:
+        (ca_cert_path_inside_pod, ca_bundle_content_or_None)
+    """
+    if not E2E_CURL_CA_CONFIGMAP:
+        return _SA_CA_PATH, None
+
+    tpl = '{{index .data "' + E2E_CURL_CA_CONFIGMAP_KEY + '"}}'
+    result = subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)